### PR TITLE
feat: side round support + banner tweaks + add tooltip

### PR DESCRIPTION
**Pull request by thelostone-mc**
## Sideround Changes

Updates models to introduce new fields on GrantCLR for mini banner.
While creating a new round , these values would have to be set right in order to ensure it's rendered as expected

- `type` -> select if GrantCLR is main / sponsor / cause
- `logo_text_hex` - > hex color of text within logo
- `banner_bg_hex` ->  hex color of background below logo (aka called banner)
- `banner_text_hex` -> hex color of text within banner
- `banner_text` -> mini text on the banner

- [x] GIVEN I am a user on the grants app homepage
WHEN I am looking at the active matching pools
THEN I see a title and description to the entire matching pool section as indicated by the design.

- [x] GIVEN I am a user on the grants app homepage
WHEN I am looking at the active matching pools
THEN I see the matching amount above "available", color-coded banners, and matching pool descriptions as indicated by the design.

- [x] GIVEN I am a Grants app internal operator on the grant brand routing policy page
WHEN I edit a grant branding policy
THEN I can edit the matching amount, hex color, and description of the matching pool banner.

##### Testing

image

----

## Tooltip Changes

As a Grant owner, I want to see how much matching I’ve received from the matching pool(s) my grant are a part of, and understand which matching pools my grant is in.

Acceptance Criteria or Definition of Done
- [x] GIVEN I am on a grant detail page
WHEN I look at the metrics under "Active Matching Rounds"
THEN I see the total matching funds the grant has received across all rounds ($X from matching pools).

- [x] GIVEN I am looking at the metrics under "Active Matching Rounds"
WHEN I hover over "$X from matching pools"
THEN I see a tooltip with a description explaining what the metric means and which matching pools the grant is a part of.

##### Testing

image

------

##### Refers/Fixes

closes https://github.com/gitcoinco/web/issues/9635
closes https://github.com/gitcoinco/web/issues/9636
closes https://github.com/gitcoinco/web/issues/9649

### CODE REVIEW

**Review by macotouenaca — APPROVED** [review]

File: app/grants/templates/grants/landing/index.html [hunk 1]
```diff
@@ -45,6 +45,17 @@
       align-items: center;
     }
 
+    .active-clr-banner {
+      height: 111px !important;
+    }
+
+    @media (max-width: 991.98px) {
```
Comment (gdixon) [inline app/grants/templates/grants/landing/index.html:52]: Could we add a utility to manage responsive fonts rather than handling them inline?

**Review by gdixon — APPROVED** [review]
LGTM! Just one small question, but we can get this reworked if we chose to go that way!

File: app/grants/models/grant.py [hunk 2]
```diff
@@ -817,6 +856,10 @@ def repr(self, user, build_absolute_uri):
 
         grant_tags = serializers.serialize('json', self.tags.all(),fields=['id', 'name'])
 
+        active_round_names= []
+        for active_round in self.in_active_clrs.all():
+            active_round_names.append(active_round.display_text)
+
```
Comment (chibie) [inline app/grants/models/grant.py:862]: would be more performant if we do

```active_round_names = self.in_active_clrs.values_list('display_text', flat=True)```

**Review by chibie — CHANGES_REQUESTED** [review]

File: app/grants/models/grant.py (hunk 2 above)
Comment (thelostone-mc) [reply app/grants/models/grant.py:862]: hmm this won't work as such ! I might have to typcast it into a list

**Pull request merged**